src/components/header.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const Header = () => (
+  <header>
+    <h1
+      style={{
+        fontSize: '6rem',
+        fontWeight: '600',
+        marginBottom: '2rem',
+        lineHeight: '1em',
+        color: '#ececec',
+        textTransform: 'lowercase',
+        textAlign: 'center',
+      }}
+    >
+      ToDos
+    </h1>
+  </header>
```

Comment:
You may consider applying all your CSS in a .CSS file instead of adding it in a module.